src/test/kotlin/ContainerUnitTest.kt

```diff
@@ -237,6 +238,60 @@ class ContainerUnitTest {
         }
     }
 
+    private val getContentFromRCCases = listOf(
+        "sng_book_various_content_rc",
+        "sng_book_various_content_rc.zip"
+    )
+
+    @Test
+    @Throws(Exception::class)
+    fun getContentFromRC() {
```

Comment:
Please create separate tests for each thing you want to test, which helps understand what is being tested, and what fails, rather than just make one generic omegatest

Reply:
Done!